src/less/link/base/link.less

```diff
@@ -41,12 +40,9 @@ button.fake-link {
     &[disabled] {
         opacity: 0.5;
     }
-}
-
-button.fake-link--action {
-    text-decoration: none;
 
     &:hover {
-        text-decoration: underline;
+        color: @link-nav-color-hover;
+        cursor: pointer;
```

Comment:
Does the cursor change only need to be on hover? 😉 (I'm aware we won't see it until it's hovered, but we set this on the element in the [pagination module](https://github.com/eBay/skin/blob/e2cb04909c35725bd5695a4f1898e355f0309d3d/src/less/pagination/base/pagination.less#L26).)